### DISCUSSION TITLE
[FEATURE REQ] Make filter actions reusable on multiple filters

### DISCUSSION THREAD

**Discussion by edifus** [autobrr/autobrr#1648] [Ideas]
**Is your feature request related to a problem? Please describe.**
Changed torrent clients and now I have to manually update the "actions" on every single filter (50+).

**Describe the solution you'd like**
Actions should be a a configured settings like, Indexer, Feeds, Clients, etc that can be used to put together a filter. Having to manually create the *exact same* "action" 50+ times is tedious.

**Describe alternatives you've considered**
No other possibilities now, just have to manually create the *exact same* action over and over and over again.

**Comment by stxmqa** [comment]
The same with "External" calls. Just had to update 20 filters to add a flag to a local script.

**Comment by mikemykhaylov** [comment]
Any progress on this? Would love to have reusable actions between filters, maybe even mass assigning them for multiple of them. Is this considered for a future roadmap feature?

**Comment by ObjectiveCesar** [comment]
+1
Right now the only way to achieve something like this is to duplicate the filter with the desired actions.
